invenio_theme_kth/views.py

```diff
@@ -1,19 +1,50 @@
 """Additional views."""
-from flask import Blueprint
+from flask import Blueprint, render_template
 from flask_login import login_required
-from invenio_app_rdm.records_ui.views.deposits import deposit_create, deposit_edit
+from invenio_app_rdm.records_ui.views.decorators import (
+    pass_draft,
+    pass_draft_community,
+    pass_draft_files,
+)
+from invenio_app_rdm.records_ui.views.deposits import (
+    get_form_config,
+    get_search_url,
+    new_record,
+)
+from invenio_rdm_records.resources.serializers import UIJSONSerializer
 
 
 @login_required
-def guarded_deposit_create(*args, **kwargs):
-    """Guard the creation page for records, based on permissions."""
-    return deposit_create(*args, **kwargs)
+@pass_draft_community
+def deposit_create_override(community=None):
+    """Create a new deposit."""
+    return render_template(
+        "invenio_app_rdm/records/deposit_override.html",
```

Comment:
When the template gets parameterized like [this](https://github.com/inveniosoftware/invenio-app-rdm/blob/1c4515b4f67b97260876a4382d9f1a7659a3b28a/invenio_app_rdm/config.py#L539) in the config, both [create ](https://github.com/inveniosoftware/invenio-app-rdm/blob/1c4515b4f67b97260876a4382d9f1a7659a3b28a/invenio_app_rdm/records_ui/views/deposits.py#L341)and [edit ](https://github.com/inveniosoftware/invenio-app-rdm/blob/1c4515b4f67b97260876a4382d9f1a7659a3b28a/invenio_app_rdm/records_ui/views/deposits.py#L360)functions should be removed.